acceptance/openstack/networking/v2/extensions/subnetpools/subnetpools_test.go

```diff
@@ -10,6 +10,21 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/subnetpools"
 )
 
+func TestCreateSubnetPool(t *testing.T) {
```

Comment:
nit: let's rename this to `TestSubnetPoolsCRUD`. For the update and delete PRs, you can just add to this test.